src/app/pages/token/token-detail/token-content/token-content.component.html

```diff
@@ -69,7 +69,7 @@
         </div>
         <div class="col-12 col-lg-3 py-3 py-lg-0 text--gray-1" *ngIf="!tokenDetail?.isNFTContract">
           <div class="caption text--gray-5 mb-1 mb-lg-0">VALUE</div>
-          <ng-container *ngIf="tokenDetail.price | gt: 0 && infoSearch['balance'] | gt: 0; else zeroValue">
+          <ng-container *ngIf="(tokenDetail.price | gt: 0) && (infoSearch['balance'] | gt: 0); else zeroValue">
```

Comment:
Chỗ này check value luôn, tương tự cho bên dưới
```suggestion
          <ng-container *ngIf="(tokenDetail.value | gt: 0); else zeroValue">
```